saleor/plugins/webhook/tasks.py

```diff
@@ -134,3 +160,27 @@ def send_webhook_request(webhook_id, target_url, secret, event_type, data):
         target_url,
         event_type,
     )
+
+
+def send_webhook_request_sync(target_url, secret, event_type, data):
```

Comment:
IMO it is the same logic as `send_webhook_request`. Why we can't extend `send_webhook_request` to return response from request by default and use it in sync logic as a normal function?
```
def trigger_webhook_sync(event_type, data, webhooks_qs=None):
    ....
    response = send_webhook_request(
            webhook.pk, webhook.target_url, webhook.secret_key, event_type, data
        )
```